### ISSUE TITLE
"Minimum bounding circle.ipynb" shows SyntaxError: only named arguments may follow *expression

### ISSUE THREAD

**Issue by ngaurav** [sjsrey/pysal#31]
<img width="621" alt="screen shot 2016-03-02 at 12 03 03 pm" src="https://cloud.githubusercontent.com/assets/12483343/13452624/8a511fbe-e06f-11e5-8791-bc3be7095e15.png">

I have looked up this issue on github. [This link](http://stackoverflow.com/a/12720593) describes it well. Also, python 3.5 accepts the code without throwing error.

**Comment by ljwolf** [comment]
Yeah, I guess I wrote this in Python 3 and did not check for 2 compat. Since there weren't any tests at the time, this is definitely an accurate diagnosis.
